redash/settings/organization.py

```diff
@@ -1,10 +1,11 @@
+from __future__ import print_function
```

Comment:
No need for this on Python2.7.

Reply:
It is useful because prevents Python 2.7 developers from adding __print__ statements that break Python 3.

Reply:
Hmm, this seems to just encourage to continue writing Python 2 code (read: dead code) 😁

How about we take this a step further (since we'll need to deal with the deprecation of [Python 2.7 next year](https://pythonclock.org/) and add a separate flake8 section that runs in a Python 3 container (until Redash switches fully to Python 3)? That way we'll see Python 2 incompatible code right away and can ask contributors to fix those bugs as part of the code review cycle.